inst/rmarkdown/templates/tuftish_ebook/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,276 @@
+---
+title: "Tuftish e-Book"
+author: "John Smith"
+date: "February 8, 2015"
+bibliography: "knitcitations.bib"
+output: 
+  tuftish_ebook:
+    latex_engine: xelatex
+    toc: true
+    toc_depth: 3
+---
+
+```{r setup,echo=FALSE,message=FALSE,cache=FALSE,include=TRUE}
+# Its a good idea to start any rmarkdown document with a block like this one to set knit defaults, setup the bibliography, and so forth.
+library(knitcitations)
+library(knitr)
+library(xtable)
+library(ggplot2)
+library(papeR)
+library(pander)
+cite_options(citation_format = "pandoc",
```

Comment:
If the cite_options and xtable options are important to the correct functioning of the template I wonder if they should become part of the format itself (we could set and reset them before/after the render). If you feel strongly that this would be better then I can even make changes in rmarkdown::render to make this possible.